be/src/Project/Web/Buyer/Buyer.Web/Areas/Products/ViewModels/Products/ProductDetailViewModel.cs

```diff
@@ -48,6 +48,7 @@ public class ProductDetailViewModel
         public string OutletTitleLabel { get; set; }
         public string ReadMoreText { get; set; }
         public string ReadLessText { get; set; }
+        public string CompletionDate { get; set; }
```

Comment:
suggestion: you should always return DateTime (UTC) from the backend and use moment like in other places to convert to local date time